sketch_obstacle_aavoidance.ino

```diff
@@ -0,0 +1,111 @@
+#include <Smartcar.h>
```

Comment:
This shouldn't have been merged. I must have missed this or it showed up in the meanwhile.

Reply:
Sorry i don't understand, can you elaborate? @platisd

Reply:
What is the point of this file? It looks like a copy of the `arduino/smartcar/smartcar.ino`

Reply:
Oh i didn't see that there was two files, i don't know what happened here. I don't know why there is copy of this file. i will remove the "sketch_obstacle_aavoidance.ino" sketch.